src/main/kotlin/com/codescene/jetbrains/services/telemetry/TelemetryService.kt

```diff
@@ -33,21 +30,19 @@ class TelemetryService() : BaseService(), Disposable {
 
         val extendedName = "${Constants.TELEMETRY_EDITOR_TYPE}/$eventName"
         // TODO: Get user ID of logged in user when authentication is implemented
-        val userId: String? = null
+        val userId: String = ""
```

Comment:
nit: you can remove explicit typing here 
```suggestion
        val userId = ""
```

Reply:
yeah, good catch...